pkg/archive/archive.go

```diff
@@ -1,15 +1,16 @@
-package util
+package archive
```

Comment:
This is properly namespaced, but it does give me discomfort that we're *already* importing the standard library's archive/tar and archive/zip, and the standard library "owns" the "archive" name. But yup, we're properly namespaced in our own world, so it should be OK. If that collision on "archive" bothers you at all maybe "archiveutil", not sure.